api/src/pcapi/alembic/versions/20231122T161119_e447c5675466_delete_ff_collective_request.py

```diff
@@ -0,0 +1,35 @@
+"""remove WIP_ENABLE_COLLECTIVE_REQUEST feature flag
+"""
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = 'e447c5675466'
+down_revision = 'd97c79fdee03'
+branch_labels = None
+depends_on = None
+
+
+def get_flag():  # type:ignore[no-untyped-def]
+    # Do not import `pcapi.models.feature` at module-level. It breaks
+    # `alembic history` with a SQLAlchemy error that complains about
+    # an unknown table name while initializing the ORM mapper.
+    from pcapi.models import feature
+
+    return feature.Feature(
+        name="WIP_ENABLE_COLLECTIVE_REQUEST",
+        isActive=False,
```

Comment:
Actuellement, ce feature flag est activé en prod. Si on re-créé ce feature flag, il faudrait qu'il ait cette même valeur.

```suggestion
        isActive=True,
```

Reply:
merci c'est modifié